src/components/AppRouter.js

```diff
@@ -30,10 +31,10 @@ function AppRouter() {
     <BrowserRouter>
       <Switch>
         <DefaultLayout exact={true} path="/" component={Splash} />
+        <DefaultLayout path="/invite_missing" component={InviteMissing} />
         <DefaultLayout path="/login" component={LogIn} />
         <DefaultLayout path="/logout" component={LogOut} />
         <DefaultLayout path="/code-of-conduct" component={CodeOfConduct} />
-        <DefaultLayout path="/me" component={() => (<Redirect to='/login' replace />)} />
```

Comment:
@rahulgi we talked about getting rid of this once we had nice buttons at all, think we're ready?

Reply:
yeah, works for me! sweet!